controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ApplicationOwnershipConfirmer.java

```diff
@@ -35,8 +37,13 @@ public ApplicationOwnershipConfirmer(Controller controller, Duration interval, J
 
     @Override
     protected void maintain() {
-        confirmApplicationOwnerships();
-        ensureConfirmationResponses();
+        try {
+            confirmApplicationOwnerships();
+            ensureConfirmationResponses();
+        }
+        catch (UncheckedIOException e) {
+            log.log(Level.INFO, () -> "IO exception handling issues, will retry: '" + Exceptions.toMessageString(e));
```

Comment:
Consider appending `". Retrying in " + maintenanceInterval()`. Same below.

Reply:
Done.